config/config.yml_example

```diff
@@ -55,6 +55,11 @@ vouch:
   # - myOrg
   # - myOrg/myTeam
 
+  tls:
```

Comment:
Please include the the env var names in the documentation

Reply:
✅